src/pages/Signup/components/SelectBox.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export default function SelectBox() {
+  return (
+    <Select name="email">
+      {EMAIL_LIST.map(list => {
+        return (
+          <option key={list.id} value={list.value}>
+            {list.email}
+          </option>
+        );
+      })}
```

Comment:
```suggestion
      {EMAIL_LIST.map(({id,value,email}) => {
        return (
          <option key={id} value={value}>
            {email}
          </option>
        );
      })}
```

key가 많지않다면 바로 구조분해할당 해서 사용해도 좋습니다.